components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/graphQL/GraphQLAPIHandler.java

```diff
@@ -106,6 +113,12 @@ public boolean handleRequest(MessageContext messageContext) {
                     OMElement body = axis2MC.getEnvelope().getBody().getFirstElement();
                     if (body != null && body.getFirstChildWithName(QName.valueOf(QUERY_PAYLOAD_STRING)) != null){
                         payload = body.getFirstChildWithName(QName.valueOf(QUERY_PAYLOAD_STRING)).getText();
+                        if (body.getFirstChildWithName(QName.valueOf("variables")) != null) {
```

Comment:
use constant for variables

Reply:
Fixed via https://github.com/wso2/carbon-apimgt/pull/12441/commits/226c49a261240dc5305f96d69aadb6f86edfae84